pkg/kubelet/rkt/container_id.go

```diff
@@ -41,6 +42,14 @@ func buildContainerID(c *containerID) kubecontainer.ContainerID {
 	}
 }
 
+func apiPodToNetPluginContainerID(pod *api.Pod) kubecontainer.ContainerID {
+	return buildContainerID(&containerID{string(pod.UID), pod.Name})
+}
+
+func runtimePodToNetPluginContainerID(pod *kubecontainer.Pod) kubecontainer.ContainerID {
+	return buildContainerID(&containerID{string(pod.ID), pod.Name})
```

Comment:
I think this doesn't actually work how we want!

`pod.ID` is of the format `rkt://$uuid`, not `$uuid`.